setup.py

```diff
@@ -19,12 +19,13 @@
     install_requires=[
         'docutils',
         'sphinxcontrib-napoleon>=0.5.1',
-        'funcsigs;python_version<"3"',
+        'funcsigs;python_version<"3.3"',
```

Comment:
For the record, I don't intend to support 3.0-3.2, but I don't have a problem with changing this.